content/en/tests/flaky_management/_index.md

```diff
@@ -116,18 +116,26 @@ Flaky Test Management uses AI to automatically assign a root cause category to e
 | **Environment Dependency** | Test depends on specific OS, library versions, or hardware. It may pass on one environment but fail on another, especially in cloud-CI environments where machines vary nondeterministically. |
 | **Unknown**             | Test is flaky for an unknown reason. |
 
+## Receive notifications
```

Comment:
Should there be actual instructions on how to enable notifications or links to the relevant Datadog pages?